### ISSUE TITLE
Parser exception when processing repository

### ISSUE THREAD

**Issue by dgarijo** [KnowledgeCaptureAndDiscovery/somef#431]
The repository numpy (https://github.com/numpy/numpy) gets the following parser error:

```
Traceback (most recent call last):
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/bin/somef", line 33, in <module>
    sys.exit(load_entry_point('somef', 'console_scripts', 'somef')())
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/lib/python3.9/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/lib/python3.9/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/lib/python3.9/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/lib/python3.9/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/dgarijo/Documents/GitHub/somef/env_3.9/lib/python3.9/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/__main__.py", line 148, in describe
    cli.run_cli(**kwargs)
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/cli.py", line 781, in run_cli
    repo_data = cli_get_data(threshold, ignore_classifiers, repo_url=repo_url,
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/cli.py", line 683, in cli_get_data
    excerpts_headers = parser_somef.extract_text_excerpts_header(unfiltered_text)
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/parser_somef.py", line 237, in extract_text_excerpts_header
    headers = extract_headers(original_text)
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/parser_somef.py", line 19, in extract_headers
    text = get_tag_content(line)
  File "/home/dgarijo/Documents/GitHub/somef/src/somef/parser_somef.py", line 450, in get_tag_content
    end = header.index("</h")
ValueError: substring not found
```

**Comment by dgarijo** [comment]
Will try to do a temporary fix for 0.8.0 capturing the exception, but this should be looked at in detail (@ma-garcia)

**Comment by dgarijo** [comment]
I dealt with the issue by adding an exception, but the issue remains when the header is distributed in several lines.
